cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -142,14 +142,24 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
                 "times. For example: --label distribution=external"
     ).associate()
 
-    private val packageManagers by option(
+    private val activatedPackageManagers by option(
         "--package-managers", "-m",
-        help = "The comma-separated package managers to activate, any of ${allPackageManagersByName.keys}."
+        help = "The comma-separated package managers to activate, any of ${allPackageManagersByName.keys}. Note that " +
+                "deactivation overrides activation."
     ).convert { name ->
         allPackageManagersByName[name]
             ?: throw BadParameterValue("Package managers must be one or more of ${allPackageManagersByName.keys}.")
     }.split(",").default(PackageManager.ALL)
 
+    private val deactivatedPackageManagers by option(
+        "--no-package-managers", "-n",
```

Comment:
`--no-package-managers` sound weird to me, using `--disable-package-managers` or `--deactivate-package-managers` make more sense to me.

Reply:
I chose a long option name that starts with "n" as I liked the short option name to be `-n`, which sounds like a "not"-operator, which actually makes some sense. And I liked that "n" alphabetically comes after "m" to resemble that the `-m` and `-n` options are related here 😀

Reply:
Understood but is it then better to change to `--not-package-managers`?

Reply:
I'd be fine with that, will change.